tests/setup_test.go

```diff
@@ -1935,3 +1936,57 @@ def main(ctx):
 		}
 	}
 }
+
+func TestUserOrgs(t *testing.T) {
+	dir, err := ioutil.TempDir("", "agola")
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	defer os.RemoveAll(dir)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	tetcd, tgitea, c := setup(ctx, t, dir)
+	defer shutdownGitea(tgitea)
+	defer shutdownEtcd(tetcd)
+
+	gwClient := gwclient.NewClient(c.Gateway.APIExposedURL, "admintoken")
+
+	_, _, err = gwClient.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: "org01", Visibility: gwapitypes.VisibilityPublic})
+	if err != nil {
+		t.Fatalf("unexpected err: %v", err)
+	}
+	_, _, err = gwClient.CreateOrg(ctx, &gwapitypes.CreateOrgRequest{Name: "org02", Visibility: gwapitypes.VisibilityPublic})
```

Comment:
You should try also with visibility Private.

Reply:
done!